resolwe_bio/processes/reads_processing/trimmomatic.yml

```diff
@@ -66,60 +78,68 @@
+    - name: trim_bases
+      label: Trim bases
+      group:
+        - name: leading
+          label: Leading quality
           type: basic:integer
           required: false
           description: >
-            This value, which should be set between 0 and 1, specifies the balance between preserving as much read length as possible vs. removal of incorrect bases. A low value of this parameter (<0.2) favours longer reads, while a high value (>0.8) favours read correctness. determined.
-    - name: leading
-      label: Leading quality
-      type: basic:integer
-      required: false
-      description: >
-        Remove low quality bases from the beginning. Specifies the minimum quality required to keep a base.
-    - name: trailing
-      label: Trailing quality
-      type: basic:integer
-      required: false
-      description: >
-        Remove low quality bases from the end. Specifies the minimum quality required to keep a base.
-    - name: crop
-      label: Crop
-      type: basic:integer
-      required: false
-      description: >
-       Cut the read to a specified length by removing bases from the end
-    - name: headcrop
-      label: Headcrop
-      type: basic:integer
-      required: false
-      description: >
-       Cut the specified number of bases from the start of the read
-    - name: minlen
-      label: Min length
-      type: basic:integer
-      required: false
-      description: >
-        Drop the read if it is below a specified length
-    - name: average_quality
-      label: Average quality
-      type: basic:integer
-      required: false
-      description: >
-        Drop the read if the average quality is below the specified level
+            Remove low quality bases from the beginning, if below a threshold quality.
+        - name: trailing
+          label: Trailing quality
+          type: basic:integer
+          required: false
+          description: >
+            Remove low quality bases from the end, if below a threshold quality.
+        - name: crop
+          label: Crop
+          type: basic:integer
+          required: false
+          description: >
+            Cut the read to a specified length by removing bases from the end.
+        - name: headcrop
+          label: Headcrop
+          type: basic:integer
+          required: false
+          description: >
+            Cut the specified number of bases from the start of the read.
+    - name: reads_filtering
+      label: Reads filtering
+      group:
+        - name: minlen
+          label: Min length
+          type: basic:integer
+          required: false
+          description: >
+            Drop the read if it is below a specified length.
+        - name: average_quality
+          label: Average quality
+          type: basic:integer
+          required: false
+          description: >
+            Drop the read if the average quality is below the specified level.
     - name: threads
```

Comment:
Remove this input and replace it with the `requirements` section of the .yml. In the process, use the value set in the `requirements` like here: https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/expression/rsem.yml#L105